tests/acceptance/test_organization_integration_external_mappings.py

```diff
@@ -101,3 +102,36 @@ def test_external_team_mappings(self):
             self.browser.click('[aria-label="Save Changes"]')
             self.browser.wait_until_not(".loading-indicator")
             self.browser.snapshot("integrations - one external team mapping")
+
+    def test_settings_tab(self):
```

Comment:
@NisanthanNanthakumar this test obviously doesn't match the name of this file or the class but I was thinking maybe we could just rename this to be the `organization_integration_configuration_tabs` or something, since they are testing the same page. thoughts?

Reply:
@MeredithAnya yeah that makes sense